scalableminds/webknossos
# Streamline password reset flow

This PR puts the token used to reset your password (send by mail) directly into the url as a query parameter. The frontend the extracts this param so that the user only has to enter the new password.

### URL of deployed dev instance (used for testing):
- [https://streamlinepasswordrestflow.webknossos.xyz](https://streamlinepasswordrestflow.webknossos.xyz)

### Steps to test:
- create a new user (with one of your real email addresses) and activate him
- log out and request to reset the password of the new user
- go to your email account and use the link to reset your password
- you do not need to enter the token yourself anymore and the resetting should still work correctly

### Issues:
- fixes #3094

------
- ~~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~~
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
